### DISCUSSION TITLE
Added badges to the new entries

### DISCUSSION THREAD

**Discussion by ajeetraina** [collabnix/awesome-testcontainers#33] [Ideas]
It would be great to add "new" badges to the recent added entries.

**Comment by mdelapenya** [comment]
Adding context from #36

@mdelapenya: https://github.com/collabnix/awesome-testcontainers/pull/36#pullrequestreview-2371484386
> My only concern is that those badges will be outdated soon, but this is expected to be reviewed and changed with certain frequency by the community, right?

@ajeetraina: https://github.com/collabnix/awesome-testcontainers/pull/36#issuecomment-2418260422
> That’s right. We will need to update it accordingly as new resources and links get introduced. We can mark the resources “new” if the created date is not more than last 4 months. WDYT?

@mdelapenya: https://github.com/collabnix/awesome-testcontainers/pull/36#issuecomment-2418486125
> Yeah, do we need to advertise that in any place? Here in the README, as a github label? Do you have an idea that could work for this?
